README.md

```diff
@@ -10,15 +10,13 @@ All supported versions are listed below. For every supported version, acceptance
 
 Supported operating systems are listed in `metadata.json` but individual releases can divert from that. For example, if Pulpcore x.y drops EL7, it will still be listed in metadata.json until all versions supported by the module have dropped it. Similarly, if x.z adds support for EL9, it'll be listed in `metadata.json` and all versions that don't support EL9 will have a note.
 
-### Pulpcore 3.15
+### Pulpcore 3.16
```

Comment:
```suggestion
### Pulpcore 3.17
```

;-)

Even tho I am not sure whether 3.16 or 3.17 should be the default. At least today Katello doesn't use 3.17 yet.

Reply:
I'd say default to the newest is good, even if Katello isn't using it.

Reply:
wfm